src/hooks/useWebSocket.js

```diff
@@ -1,84 +1,82 @@
+import { getIsLoggedIn, getToken } from '@/redux-store/selectors';
 import { useRef, useEffect } from 'react';
+import { useSelector } from 'react-redux';
 import { useStompClient } from 'react-stomp-hooks';
 
 export const useWebSocket = () => {
+  const isUserLoggedIn = useSelector(getIsLoggedIn);
+  const token = useSelector(getToken);
```

Comment:
Are this two selectors nessessary here? We have checking in the WebsocketProvider